pkg/clustermanager/cluster_manager.go

```diff
@@ -523,6 +523,22 @@ func (c *ClusterManager) getWorkloadClusterKubeconfig(ctx context.Context, clust
 	return nil
 }
 
+// CreateEKSAReleaseBundle applies the eks-a release bundle to the cluster.
+func (c *ClusterManager) CreateEKSAReleaseBundle(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error {
+	if clusterSpec.Cluster.Namespace != "" {
+		if err := c.clusterClient.CreateNamespaceIfNotPresent(ctx, cluster.KubeconfigFile, clusterSpec.Cluster.Namespace); err != nil {
+			return err
+		}
+	}
+
+	clusterSpec.Cluster.AddManagedByCLIAnnotation()
```

Comment:
what is this for?

Reply:
This condition let's the webhook create management clusters using the eksa controller.

Reply:
ah good point
then could we set it before applying the cluster spec? so in a different task, whichever uses the cluster creator

Also maybe add a comment since not even me remember what this was for 🤦🏻

Reply:
In that case I'll add it before the create workload cluster task and remove it from this PR.